docs/content/latest/admin/yb-ts-cli.md

```diff
@@ -0,0 +1,208 @@
+
+```sh
+$ ./bin/yb-ts-cli
+```
+
+## Commands
+
+##### are_tablets_running
+
+If all tablets are running, returns "All tablets are running".
+
+**Syntax**
+
+```sh
+yb-ts-cli [ --server_address=<addr> ] are_tablets_running
+```
+
+##### count_intents
+
+Print the count of uncompleted intents (or [provisional records](../../../architecture/transactions/ditributed-txns/#provisional-records)). Useful for debugging and troubleshooting.
+
+**Syntax**
+
+```sh
+$ ./bin/yb-ts-cli  [ --server_address=<addr> ] count_intents
+```
+
+##### current_hybrid_time
+
+Prints the value of the current [hybrid time](../../../architecture/transactions/single-row-transactions/#hybrid-time-as-an-mvcc-timestamp).
+
+**Syntax**
+
+```sh
+$ ./bin/yb-ts-cli  [ --server_address=<addr> ] current_hybrid_time
+```
+
+##### delete_tablet
+
+Deletes the tablet with the specified tablet ID (`tablet_id`) and reason.
+
+**Syntax**
+
+```sh
+$ ./bin/yb-ts-cli  [ --server_address=<addr> ] delete_tablet <tablet_id> "<reason-string>"
+```
+
+- *tablet_id*: The identifier (ID) for the tablet.
+- *reason-string*: Text string noting why the tablet was deleted.
+
+##### dump_tablet
+
+Dump, or export, the specified tablet ID (`tablet_id`).
+
+**Syntax**
+
+```sh
+yb-ts-cli [ --server_address=<addr> ] dump_tablet <tablet_id>
+```
+
+- *tablet_id*: The identifier (ID) for the tablet.
+
+##### list_tablets
+
+Lists the tablets on the specified tablet server, displaying the following information:
+
+- Tablet id
+- State
+- Table name
+- Partition
+- Schema
+- properties
+  - lumn_name
+  - Tablet id
+  - State
+  - Table name
+  - Partition
+  - Schema
+
+**Syntax**
+
+```sh
+yb-ts-cli [ --server_address=<addr> ] list_tablets
+```
+
+##### set_flag
+
+Sets the specified configuration option (flag) for the tablet server.
+
+**Syntax**
+
+```sh
+$ ./bin/yb-ts-cli set_flag [ --force ] <flag> <value>
+```
+
+- --force: Optional. See [--force](#force).
+- *flag*: The `yb-tserver` configuration option (without the `--`) to be set. See [`yb-tserver`](../../reference/configuration/yb-tserver/#configuration-options)
+- *value*: The value to be applied.
+
+##### status
+
+Prints the status of the tablet server, including information on the node instance, bound RPC addresses, bound HTTP addresses, and version information.
+
+**Syntax**
+
+```sh
+$ ./bin/yb-ts-cli  [ --server_address=<addr> ] status
+```
+
+For an example, see [Return the status of a tablet server](#return-the-status-of-a-tablet-server)
+
+## Options
+
+The following options (or flags) can be used, when specified, with the commands above.
+
+##### --force
+
+If `true`, allows the [`set_flag`](#set-flag) command to set an option which is not explicitly marked as runtime-settable. The change may be ignored on the server or may cause the server to crash.
+
+Default: `false`
+
+##### --server-address
+
+The address of the tablet server to run against.
```

Comment:
as per above, this is host:port and is 9100 for TS and 7100 for master (some operations work on master too, such as setting flags)

It might be that it defaults to the TS rpc port of 9100